src/components/v2/values/chartValuesDiff/ChartValuesView.tsx

```diff
@@ -1302,18 +1302,19 @@ function ChartValuesView({
                             />
                         )}
                         <div className="chart-values-view__hr-divider bcn-1 mt-16 mb-16" />
-                        {!isDeployChartView && commonState.showRepoSelector && (
-                            <ChartRepoSelector
-                                isExternal={isExternalApp}
-                                isUpdate={!!isUpdate}
-                                installedAppInfo={commonState.installedAppInfo}
-                                handleRepoChartValueChange={handleRepoChartValueChange}
-                                repoChartValue={commonState.repoChartValue}
-                                chartDetails={commonState.repoChartValue}
-                                showConnectToChartTippy={commonState.showConnectToChartTippy}
-                                hideConnectToChartTippy={hideConnectToChartTippy}
-                            />
-                        )}
+                        {!isDeployChartView &&
```

Comment:
Could you please add the supporting comment

Reply:
Done